scripts/qa_crowbarsetup.sh

```diff
@@ -1927,7 +1943,13 @@ function onadmin_addupdaterepo()
 {
     pre_hook $FUNCNAME
 
-    local UPR=/srv/tftpboot/repos/Cloud-PTF
+    onadmin_set_source_variables
+
+    if iscloudver 5plus; then
```

Comment:
Can you tell me where to introduce the variable? I'm not familiar enough with the script :/